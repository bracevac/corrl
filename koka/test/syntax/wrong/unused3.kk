function main() {
  val xs = [1,2,3]
  1 xs.length
}