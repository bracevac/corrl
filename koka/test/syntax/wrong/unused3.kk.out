test/syntax/wrong/unused3.kk(3, 3): error: expression has no effect and is unused
 hint: did you forget an operator? or is there a space between an application?